plugins/git-release-manager/src/features/Stats/Row/RowCollapsed/RowCollapsed.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { Box } from '@material-ui/core';
+
+import { ReleaseStats } from '../../contexts/ReleaseStatsContext';
+import { ReleaseTagList } from './ReleaseTagList';
+import { ReleaseTime } from './ReleaseTime';
+
+interface RowCollapsedProps {
+  releaseStat: ReleaseStats['releases']['0'];
+}
+
+export function RowCollapsed({ releaseStat }: RowCollapsedProps) {
+  return (
+    <Box
+      margin={1}
+      style={{
+        display: 'flex',
+        alignItems: 'stretch',
+        paddingLeft: '10%',
+        paddingRight: '10%',
```

Comment:
just gotta say that Box has a surprising number of these styles available as props too

Reply:
Does the prop alternative provide any benefit? 🤔

Reply:
I mean, not all that much I guess. There's typescript safety in both cases so ...

Reply:
Unless it's things that play into material-ui's design system, such as `margin={x}` where `x` gets translated to a normalized value, I think `style` looks a bit more straight forward 🙂